primitives/src/lib.rs

```diff
@@ -154,3 +155,8 @@ pub mod nft {
 		}
 	}
 }
+
+/// Used in the LBP config to perform some action when the liquidity is removed from a pool.
+pub trait OnRemoveLbpLiquidity<AccountId, AssetId, Balance> {
```

Comment:
this is not really how you use this handler, it is named very generically, but you only use it specifically to transfer portion of tokens from pool

therefore it should be named:
`TransferLiquidityToPool`
`transfer_liquidity_to_pool`